boxsdk/auth/jwt_auth.py

```diff
@@ -360,3 +361,51 @@ def _normalize_rsa_private_key_passphrase(passphrase):
                 .format(passphrase.__class__.__name__)
             )
         return passphrase
+
+    @classmethod
+    def from_config_dictionary(cls, config_dictionary):
```

Comment:
Nitpick, but I somewhat lean towards "settings" over "config". Since the JSON dictionary has a top-level name "boxAppSettings", and the admin console section is named "App Settings".

I'm not totally opposed to "config" though.

Reply:
This method, and also `from_config_file`, should accept `**kwargs`.

We still want users of these constructors to be able to, e.g. specify their own network layer, or specify a different algorithm.